File: 1° Período - Introdução a Programação 1/Exercicios em Python/Lista 5 - Python/q9.py
#Escrever um algoritmo que leia o nome e o sexo de 50 pessoas e informe o nome e o sexo de cada
#uma. Opções: (ex.: 1 – Masculino / 2 - Feminino). No final informe total de homens e de mulheres.

homem=0
mulher=0
for i in range(5):
    nome=input("Nome: ")
    sexo=int(input("1 - Masculino\n2 - Feminino\nOpção: "))
    print("NOME: ", nome)
    if sexo==1:
        print("SEXO: Masculino\n")
        homem=homem+1
    if sexo==2:
        print("SEXO: Feminino\n")
        mulher=mulher+1
   
print("\nTotal de homens: ", homem, "\nTotal de mulheres: ", mulher)

File: 1° Período - Introdução a Programação 1/Exercicios em C/Lista 5 - C/q9.c
//Escrever um algoritmo que leia o nome e o sexo de 50 pessoas e informe o nome e o sexo de cada
//uma. Opções: (ex.: 1 – Masculino / 2 - Feminino). No final informe total de homens e de mulheres.
#include<stdio.h>
int main(){
	int sexo, homem, mulher, i;
	homem=0;
	mulher=0;
	for(i=0; i<5; i++){
		printf("1 - Masculino\n2 - Feminino\nOpção: ");
		scanf("%d", &sexo);
		if(sexo==1){
	        printf("SEXO: Masculino\n");
	        homem=homem+1;
	    }
	    else if(sexo==2){
	    	printf("SEXO: Feminino\n");
	        mulher=mulher+1;
	    }
	}
printf("\nTotal de homens: %d\nTotal de mulheres: %d", homem, mulher);	
}
